src/items.rs

```diff
@@ -1989,11 +1994,13 @@ fn rewrite_static(
     static_parts: &StaticParts<'_>,
     offset: Indent,
 ) -> Option<String> {
+    println!("rewriting static");
```

Comment:
You left in a debug print, it seems.

Reply:
I am surprised that this did not make any of the tests fail...?

Reply:
> I am surprised that this did not make any of the tests fail...?

same (https://github.com/rust-lang/rustfmt/issues/6210#issuecomment-2186797085)

we'll do a postmortem of sorts to review and hopefully identify some preventative measures to prevent a recurrence